app/api/v2/atbds.py

```diff
@@ -282,7 +282,9 @@ def get_upload_url(
     upload_path = Path(atbd_id) / "uploads"
     file_name = f"atbd_{atbd_id}_{int(datetime.datetime.now().timestamp())}.pdf"
     file_path = upload_path / file_name
-    pdf_upload = PDFUpload(atbd_id=atbd_id, file_path=str(file_path), created_by=user.sub)
+    pdf_upload = PDFUpload(
+        atbd_id=atbd_id, file_path=str(file_path), created_by=user.sub
+    )
     db.add(pdf_upload)
     db.commit()
     db.refresh(pdf_upload)
```

Comment:
Ideally, we should move this to `crud/uploads`

Reply:
Sure. Updated to use crud
https://github.com/NASA-IMPACT/nasa-apt/blob/3a15b7bafe40ca1f212316b39b4654e15037cb3d/app/api/v2/atbds.py#L283-L289